exercises/complex-numbers/canonical-data.json

```diff
@@ -68,7 +68,8 @@
     {
       "description": "Imaginary unit",
       "property": "mul",
```

Comment:
I put more general properties first and the special cases afterwards. But since i^2=-1 is so fundamental, it would make sense to have it as the very first test, indeed.